core/src/components/split-pane/test/multiple/index.html

```diff
@@ -0,0 +1,69 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+  <head>
+    <meta charset="UTF-8" />
+    <title>Split Pane - Multiple</title>
+    <meta
+      name="viewport"
+      content="width=device-width, initial-scale=1.0, minimum-scale=1.0, maximum-scale=1.0, user-scalable=no"
+    />
+    <link href="../../../../../css/ionic.bundle.css" rel="stylesheet" />
+    <link href="../../../../../scripts/testing/styles.css" rel="stylesheet" />
+    <script src="../../../../../scripts/testing/scripts.js"></script>
+    <script type="module" src="../../../../../dist/ionic/ionic.esm.js"></script>
+  </head>
+  <body>
+    <ion-split-pane id="pane-one" content-id="content-one">
+      <ion-menu id="pane-one-menu-one" side="start" content-id="content-one">
+        <div class="ion-padding">Split One Menu Two</div>
+      </ion-menu>
+
+      <div class="ion-page" id="content-one">
+        <ion-content class="ion-padding">
+          Page Content One
+          <button id="show-pane-two" onclick="showSecondPane()">Show Second Pane</button>
+        </ion-content>
+      </div>
+
+      <ion-menu id="pane-one-menu-two" side="end" content-id="content-one">
+        <div class="ion-padding">Split One Menu Two</div>
+      </ion-menu>
+    </ion-split-pane>
+
+    <ion-split-pane id="pane-two" content-id="content-two" disabled="true" class="ion-page-hidden">
```

Comment:
Unsure if this is related to this PR, but there seems to be something strange going on with the menus when trying to open them with gestures. At least one of the four menus can't be opened, and sometimes leads to errors in the console while dragging. It seems inconsistent which one(s); I tried a few more refreshes after the below recording and got different results.

https://github.com/ionic-team/ionic-framework/assets/90629384/a87a5903-f753-45cb-a522-3bb6bddc9071

Reply:
Yeah that's this issue: https://github.com/ionic-team/ionic-framework/issues/19410

The menus all get added to the same document, so there's an inconsistency in the order in which they open because it's dependent on which menus registers first. For example, if you refresh a few times you'll eventually find that the menu you could open before no longer opens. In reality, a menu that is hidden is being activated instead. However, if that menu has `display: none` then it will have a width of 0, resulting in the animation error (because we're dividing by 0 when computing the gesture step).